ktor-http/ktor-http-cio/api/ktor-http-cio.api

```diff
@@ -279,22 +283,30 @@ public final class io/ktor/http/cio/websocket/DefaultWebSocketSessionKt {
 	public static final fun DefaultWebSocketSession (Lio/ktor/http/cio/websocket/WebSocketSession;JJ)Lio/ktor/http/cio/websocket/DefaultWebSocketSession;
 }
 
+public abstract interface annotation class io/ktor/http/cio/websocket/ExperimentalWebSocketExtensionApi : java/lang/annotation/Annotation {
+}
+
 public abstract class io/ktor/http/cio/websocket/Frame {
 	public static final field Companion Lio/ktor/http/cio/websocket/Frame$Companion;
-	public synthetic fun <init> (ZLio/ktor/http/cio/websocket/FrameType;[BLkotlinx/coroutines/DisposableHandle;Lkotlin/jvm/internal/DefaultConstructorMarker;)V
+	public synthetic fun <init> (ZLio/ktor/http/cio/websocket/FrameType;[BLkotlinx/coroutines/DisposableHandle;ZZZLkotlin/jvm/internal/DefaultConstructorMarker;)V
```

Comment:
breaking change?

Reply:
Nope, the class is sealed and can't be used explicitly